app/controllers/experiments/list.js

```diff
@@ -8,7 +8,7 @@ export default Ember.Controller.extend({
     isShowingModal: false,
     breadCrumb: 'Experiments',
     queryParams: ['sort', 'match', 'state', 'q'],
-    state: null,
+    state: 'All',
```

Comment:
Also, where can I find a clear, central definition for the allowed values of the `state` parameter?

(this is always important to future developers, but even more so if the parameter might be case-sensitive)